gamemodes/zombieplague/gamemode/languages/ukrainian.lua

```diff
@@ -65,7 +65,8 @@ local Language = {PrettyName = "Українська",
 			ExtraItemExtraAmmoName = "Додаткові набої",
 			RoundsLeft = "%d раундів залишилося!",
 			FinalRound = "Фінальний раунд!",
-			SaveSelection = "Зберегти вибір?"
+			SaveSelection = "Зберегти вибір?",
+			RandomSelection = "Випадкова зброя?"
```

Comment:
That's interessing

Reply:
I know

Reply:
I was thinking: Ok, we have this, but it's not implemented